src/ontodia/diagram/elementViews.ts

```diff
@@ -51,6 +53,12 @@ export class UIElementView extends joint.dia.ElementView {
             parent: d3.select($root.get(0)),
             captionText: '<loading...>',
         });
+        this.image = new svgui.Image({
+            parent: this.box.root,
+            size: { width: 100, height: 30 },
```

Comment:
The braces of object literals should not contain spaces next to them (see STYLEGUIDE).